test_simple_pe/test_simple_pe_magma.py

```diff
@@ -86,10 +86,6 @@ class SimplePETester(ResetTester, ConfigurationTester):
 
     m.compile("test_simple_pe/build/pe", pe, output="coreir",
               passes=["rungenerators", "flatten", "cullgraph"])
-    # For some reason cullgraph above doesn't result in a culled output,
```

Comment:
was this just fixed in coreir/pycoreir?

Reply:
Nope, blocked by https://github.com/rdaly525/coreir/issues/632

Reply:
But it doesn't affect the test. Cullgraph just cleaned up the output join by removing unused definitions. This was nice for debugging (sending the json over to Cristian), but not necessary for the test functionality.

Reply:
Oh, but the fact that cull graph isn't doing it initially is also still an issue. I'll file an issue in coreir.